src/com/github/johnthagen/cppcheck/Configuration.java

```diff
@@ -13,16 +13,27 @@
   private boolean modified = false;
   private JFilePicker cppcheckFilePicker;
   private JTextField cppcheckOptionsField;
+  private JFilePicker cppcheckMisraFilePicker;
   private static final String CPPCHECK_NOTE =
     "Note: C++ projects should leave --language=c++ appended to the cppcheck options to avoid some " +
     "false positives in header files due to the fact that cppcheck implicitly defaults to " +
     "setting --language to \"c\" for .h files.\n\n" +
     "You should not include any --template={} in the options.";
+  private static final String CPPCHECK_MISRA_NOTE =
+    "Using MISRA requires a rule texts file, which can be obtained from MISRA themselves (Their license prohibits distributing the rules texts)\n\n" +
+    "Create a .json file near your CppCheck installation and point to it here\n" +
```

Comment:
Please change all instances of `CppCheck` to `Cppcheck`. Please see: https://github.com/johnthagen/clion-cppcheck/commit/0e9d4fc32957fb603405ece7e61193c7bb2ddb14#commitcomment-38153765

Reply:
Changed as requested.